docs/docs-content/byoos/usecases/vmware/konvoy.md

```diff
@@ -0,0 +1,465 @@
+        ```
+
+    {/* prettier-ignore */}
+    <details>
+    <summary>Complete FIPS configuration file</summary>
+
+    ```yaml {16-19}
+    - name: "redhat 8 fips: ensure fips_enabled is 1"
+      lineinfile:
+        dest: /proc/sys/crypto/fips_enabled
+        line: "1"
+        state: present
+      become: yes
+      check_mode: yes
+      register: sysfips
+
+    - name: "redhat 8 fips: run fips enable command"
+      command: fips-mode-setup --enable
+      when:
+        - sysfips is changed
+
+    - name: "redhat 8 fips: reboot to pick up fips changes"
+      ansible.builtin.reboot:
+        post_reboot_delay: 120
+        pre_reboot_delay: 30
+        connect_timeout: 120
+      when:
+        - sysfips is changed
+    ```
+
+    </details>
+
+    :::tip
+
+    Additional FIPS configuration can be found in the **overrides/fips.yaml** and **overrides/fips-configure.yaml**
+    files.
+
+    :::
+
+9.  Start the Packer build process by issuing the following command.
+
+    ```shell
+    ./konvoy-image build images/ova/rhel-88.yaml \
+    --extra-vars ansible_sudo_pass="admin" \
+    --overrides=overrides/fips.yaml \
+    --overrides=overrides/fips-configure.yaml
+    ```
+
+    The build process will take some time to complete. Once the build process is complete, you will have a FIPS
+    compliant custom RHEL with Konvoy image located in the vSphere datastore folder you specified in the Packer
+    configuration file. Take note of the image ID.
+
+    ```shell hideClipboard {11}
+    ==> vsphere-clone.kib_image: Convert VM into template...
+    vsphere-clone.kib_image: Closing sessions ...
+    ==> vsphere-clone.kib_image: Running post-processor: packer-manifest (type manifest)
+    ==> vsphere-clone.kib_image: Running post-processor:  (type shell-local)
+    ==> vsphere-clone.kib_image (shell-local): Running local shell script: /tmp/packer-shell2158067181
+    Build 'vsphere-clone.kib_image' finished after 18 minutes 44 seconds.
+
+    ==> Wait completed after 18 minutes 44 seconds
+
+    ==> Builds finished. The artifacts of successful builds are:
+    --> vsphere-clone.kib_image: konvoy-rhel-88-fips-kernel-1.27.6+fips.0-20240301141207
+    --> vsphere-clone.kib_image: konvoy-rhel-88-fips-kernel-1.27.6+fips.0-20240301141207
+    --> vsphere-clone.kib_image: konvoy-rhel-88-fips-kernel-1.27.6+fips.0-20240301141207
+    ```
+
+</TabItem>
+</Tabs>
+
+## Validate
+
+1. Log in to the vSphere web client and navigate to the datastore folder where the custom RHEL with Konvoy image is
+   located. Verify that the image is present in the folder.
+
+2. Create a cluster profile and use the BYOOS pack to reference the custom RHEL with Konvoy image. Specify the path to
+   the custom RHEL with Konvoy image in the pack's YAML file. Refer to
+   [Usage with Cluster Profile](#usage-with-cluster-profile) section for more details.
+
+3. Deploy a cluster using the cluster profile you created.
+
+## Usage with Cluster Profile
```

Comment:
Should this section not have its own separate validation steps?

Reply:
I didn't want to create a how-to as we already have how-tos that explain how to create a cluster profile. So I wrote something that speaks to the flow at a high level.